capacity4more/behat/features/group_status.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Group Status
+  Test group status field
+
+  @api
+  Scenario: Check requested group as admin
+    Given I am logged in as user "mariecurie"
+    When  I start editing group "Requested group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "archived"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+
+  @api
+  Scenario: Check requested group as group owner
```

Comment:
@zero2one But he can change status for requested to "deleted" due to schema. And due to the Group Rights table group owner can view group and change group properties (incl. group status -> to deleted as I understood).